public/API/API_GetUserProfile.php

```diff
@@ -0,0 +1,55 @@
+<?php
+
+/*
+ *  API_GetUserProfile
+ *    u : username
+ *
+ *  string     User                    name of user
+ *  int        ID                      unique identifier of the user
+ *  int        TotalPoints             number of hardcore points the user has
+ *  int        TotalSoftcorePoints     number of softcore points the user has
+ *  int        TotalTruePoints         number of "white" points the user has
+ *  int        Permissions             unique identifier of user's account type
+ *  datetime   MemberSince             when the user joined the site
+ *  int        Untracked               "1" if the user is untracked, otherwise "0"
+ *  string     UserPic                 site-relative path to the user's profile picture
+ *  string     Motto                   the user's motto
+ *  int        UserWallActive          "1" if the user allows comments to be posted to their wall, otherwise "0"
```

Comment:
Is `UserWallActive` really important to the caller? It's not like they can see the user wall or append to it.

Reply:
I was wondering the same, however it is included in the summary, too. Might as well stay consistent in this case?
It shouldn't be included in the next iteration of the API imo.